src/zx/FunctionalityConstruction.cpp

```diff
@@ -333,13 +350,14 @@ namespace zx {
         return diag;
     }
     bool FunctionalityConstruction::transformableToZX(const qc::QuantumComputation* qc) {
-        return std::all_of(qc->cbegin(), qc->cend(), [](const auto& op) { return transformableToZX(op.get()); });
+        return std::all_of(qc->cbegin(), qc->cend(), [&](const auto& op) { return transformableToZX(op.get()); });
     }
 
     bool FunctionalityConstruction::transformableToZX(qc::Operation* op) {
```

Comment:
could `op` here technically also be a `const` pointer?

Reply:
A truly sneaky const. Forgot that you can cast to const pointer, I think.